### ISSUE TITLE
Update installation insctructions

### ISSUE THREAD

**Issue by CedCannes** [M6Web/DaemonBundle#23]
Installation instructions needs an update to work with my app using Symfony 2.8.8.
The file M6WebDaemonBundle.php required by my AppKernel is appears only since V 1.2.0.

## Installation (that works for me)

Via composer :

```json
"require": {
    "m6web/daemon-bundle":"1.4.*"
}
```

then enable the bundle in your kernel:

```php
<?php

$bundles = [
    new M6Web\Bundle\DaemonBundle\M6WebDaemonBundle()
];
```

**Comment by Oliboy50** [comment]
I think, this will be fixed with #21 :)

**Comment by CedCannes** [comment]
Sure, I hadn't seen this PR sorry.
